.lldbinit

```diff
@@ -0,0 +1 @@
+command script import ./tools/lldbhalide.py
```

Comment:
I don't think putting this in the root of the tree is the right thing given how lldb works. This is unlikely to be the current working directory when lldb is invoked. If it just did the right thing I'd be ok with it, but given one is probably going to have to give some arguments to lldb anyway, I think it should go in scripts or a new directory to avoid clutter in the root. It probably also does not need to be named as a dot file in this case. One would likely source the file from the main lldbinit. I could also see adding support for copying it into the test bin directories if there were a way to automatically run the test binary under lldb in the build support.

Reply:
CLion invokes LLDB from the project root, which is why I created this file in the first place.

Reply:
I don't really want to put this in my `~/.lldbinit` because it could interfere with other projects.